### PR TITLE
Removed #include JDastro in the macro exampleJDdarkmatter and changed <> with "" in JDDarkmatter.h

### PR DESCRIPTION
**Pull request by micheledoro**

### CODE REVIEW

File: source/JDDarkMatter.h [hunk 1]
```diff
@@ -30,163 +33,159 @@ using namespace std;
...
+  
+  
+  ///////////////////////////////////////////////////////
+  //Bool_t
+  ///////////////////////////////////////////////////////
+  Bool_t bIsBonnivard;
+  Bool_t bIsGeringer;
+  Bool_t bIsJFactor;
+  Bool_t bIsJFactorSigma1;
 };
```
Comment (Jopana) [inline source/JDDarkMatter.h:190]: estas usando 'emacs' verdad? Yo usaba 'eclipse' y ya habia visto que habia este problema con display... Todos los tabs se desajustan... 

Tenia pensado seguir usando eclipse ahora... tenemos que ver si este problema persiste...

File: macros/exampleJDDarkMatter.cxx [hunk 2]
```diff
@@ -26,12 +26,11 @@
 #include <iostream>
 #include <TStyle.h>
 
-#include "../source/JDDarkMatter.cc"
-#include "../source/JDAstroProfile.cc"
+#include "/Users/mdoro/Soft/ObservationOptimization/source/JDDarkMatter.cc"
+//#include "/Users/mdoro/Soft/ObservationOptimization/source/JDAstroProfile.cc"
```
Comment (Jopana) [inline macros/exampleJDDarkMatter.cxx:30]: No deberiamos hacer commits con paths locales no?

Ademas... no entiendo esta parte del commit. Este fichero no ha cambiado en nada no?

**Comment by micheledoro** [comment]
Please do not accept this suggestions. I will move to eclipse myself. Do no accept any commit until now. They are minor

**Comment by Jopana** [comment]
We move all the discussion and changes to the new 'CTA_paper' branch.

https://github.com/IndirectDarkMatterSearchesIFAE/ObservationOptimization/tree/CTA_paper

**Pull request closed without merging**